packages/database/src/models/UserPatientView.ts

```diff
@@ -0,0 +1,55 @@
+import { DataTypes } from 'sequelize';
+
+import { SYNC_DIRECTIONS } from '@tamanu/constants';
+
+import { Model } from './Model';
+import type { InitOptions, Models } from '../types/model';
+
+export class UserPatientView extends Model {
+  declare id: string;
+  declare viewedById: string;
```

Comment:
```suggestion
  declare accessedById: string;
```
Or just 
```suggestion
  declare userId: string;
```
(honestly prefer the second one because it goes with the title of the table/model)

Reply:
Yep i have an pr coming in to this soon with all the discussed changes from past few days